src/ontology/edits/oeo-shared.omn

```diff
@@ -602,7 +828,31 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/869"@en,
         rdfs:label "study"@en
     
     SubClassOf: 
-        OEO_00000340
+        OEO_00000340,
+        
+            Annotations: <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/473
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/557"
+        OEO_00000509 some (<http://purl.obolibrary.org/obo/RO_0000087> some OEO_00090001)
+    
+    
+Class: OEO_00020012
```

Comment:
Btw, It is possible to split class definitions and axioms into different modules, see [here](https://github.com/OpenEnergyPlatform/ontology/pull/748#discussion_r635958298). 
Would it be an option to leave `study report` in oeo-model and define the `OEO_00000506 some (<http://purl.obolibrary.org/obo/RO_0000087> some OEO_00000064)` and `<http://purl.obolibrary.org/obo/RO_0002353> some OEO_00020011` there, too? Or do I oversee something?
I think, we should only move classes to oeo-sheared, if it's relly necessary...

Reply:
I haven't checks yet, if this would also be an option for `funder`, `sponsor` and `organisation`.

Reply:
I think there are two conflicting interests:
* One interest is to have the definitions of classes in the module where they fit thematically best.
* A second interest is to have the entities self-contained with all axioms in one place so that _one_ code block gives all information about an entity.

I don't know what is more important, but at least my preference is the latter.

@OpenEnergyPlatform/oeo-general-expert-formal-ontology : Is there a preferred option from a formal ontological point of view? And what is better for reusability of the ontology?

Reply:
Bilateral decision with @l-emele: leave the PR as is and discuss this question in next dev-meeting.